e2e-tests/TestHelper.ts

```diff
@@ -0,0 +1,28 @@
+import { BrowserUtils } from 'wdio-allure-ts';
+
+/**
+ * Holds common methods for tests
+ */
+export const sampleAppUrl: string = 'http://127.0.0.1:8000';
+/**
+ * common describe for specs
+ * @param name spec name
+ * @param body spec body -placeholder
+ */
+
+export function describeCommon(name: string, body: () => any): void {
```

Comment:
I prefer not extending the describe (describeCommon), 
I think this adds another layer of complexity people don't usually expect. 

can we instead inject this as a global?

this looks like it's running on mocha, so maybe this can help
https://stackoverflow.com/questions/10561598/global-before-and-beforeeach-for-mocha


Another solution is to export a function with the beforeEach, and let every test import and run it (slightly more verbose than `describeCommon`, but a bit more explicit)

```
export function globalBeforeEach(() => {
    beforeEach(() => {
        BrowserUtils.navigateToUrl(sampleAppUrl);
        BrowserUtils.waitForDisplayed("//*[@id='top']");
    });
})

```


And then in a test, i'll look like this

```

describe('foo', () => {
   globalBeforeEach()

   it('runs a test', () = {})

})
```

Reply:
@patrick-tolosa I think this is a subjective style preference, I think the existing approach is good, I like it's simplicity.
This file is copied from wdio-allure-ts repo: https://github.com/cloudinary/wdio-allure-ts/blob/master/src/test/TestHelper.ts
Please feel free to do a refactor pr or add this as a task.

Reply:
Fair enough